test/lint/eslint/esm/.eslintrc.cjs

```diff
@@ -3,5 +3,11 @@
 module.exports = {
     parserOptions: {
         sourceType: 'module'
-    }
+    },
+    overrides: [
+        {
+            files: ['*.cjs'],
+            parserOptions: { sourceType: 'script' }
```

Comment:
This was necessary for the `should lint ESM files` test - now that `cjs` is also being scanned, the test needed an update, but it was also complaining that `use strict` is not necessary, because it was getting parsed as a module. TBH, I didn't have time to properly figure out why - eslint should be doing this by default for `*.cjs` files...

But this does unblock further improvements, I'd hope.